docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -147,13 +147,14 @@ setting namespaces (broker, peon, historical)
 |--------|-----------|-------|
 |`druid.lookup.namespace.cache.type`|Specifies the type of caching to be used by the namespaces. May be one of [`offHeap`, `onHeap`]. `offHeap` uses a temporary file for off-heap storage of the namespace (memory mapped files). `onHeap` stores all cache on the heap in standard java map types.|`onHeap`|
 |`druid.lookup.namespace.numExtractionThreads`|The number of threads in the thread pool dedicated for lookup extraction and updates. This number may need to be scaled up, if you have a lot of lookups and they take long time to extract, to avoid timeouts.|2|
+|`druid.lookup.namespace.numBufferEntries`|If using offHeap caching, the number of records to be stored on an on-heap buffer|100,000|
 
 The cache is populated in different ways depending on the settings below. In general, most namespaces employ 
 a `pollPeriod` at the end of which time they poll the remote resource of interest for updates.
 
 `onHeap` uses `ConcurrentMap`s in the java heap, and thus affects garbage collection and heap sizing.
-`offHeap` uses a 10MB on-heap buffer and MapDB using memory-mapped files in the java temporary directory.
-So if total `cachedNamespace` lookup size is in excess of 10MB, the extra will be kept in memory as page cache, and paged in and out by general OS tunings.
+`offHeap` uses an on-heap buffer and MapDB using memory-mapped files in the java temporary directory.
+So if total number of entries in the `cachedNamespace` is in excess of the buffer's configured capacity, the extra will be kept in memory as page cache, and paged in and out by general OS tunings.
```

Comment:
Why cachedNamespace is in ticks? To what does it refer?

Reply:
It highlights the text in the generated docs, there looks to be the convention that when a string in the docs references a key or value in the configuration it's highlighted in this way.  

Since cachedNamespace is a value for the lookup type it's highlighted here & in other places where it's mentioned e.g. the "Example Configuration" section.